volcli/volcli.go

```diff
@@ -120,13 +120,14 @@ func globalUpload(ctx *cli.Context) (bool, error) {
 		return false, err
 	}
 
-	cfg, err := config.NewClient(ctx.GlobalString("prefix"), ctx.GlobalStringSlice("etcd"))
+	resp, err := http.Post(fmt.Sprintf("http://%s/global", ctx.String("volmaster")), "application/json", bytes.NewBuffer(content))
```

Comment:
minor: we should perhaps use a function to form the request URL (i.e. wrap `fmt.Sprintf` there), that way if we later change something (like url scheme from http to https, or volmaster name to apiserver or similar) we won't need to again touch all these places.